templates/frontend/components/skipLinks.tpl

```diff
@@ -8,7 +8,7 @@
  * Skip links to aid navigation when tabbing for screen reader accessibility
  *}
 
- <nav class="cmp_skip_to_content">
+ <nav class="cmp_skip_to_content" aria-label="Jump to content links">
```

Comment:
This needs to be a translatable string. So `{translate key="..."}` and then set up the locale key.